### PR TITLE
About us

### PR DESCRIPTION
**Pull request by sajaabusafia**

### CODE REVIEW

File: aboutus.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./aboutus.css">
+    <title>About Us</title>
+</head>
+<body>
+    <iframe class="header" src="index.html" frameborder="0"></iframe>
+    <div class="about">
+
+    <div class="about_Background_img">
+        <img src="images/background.jpg" alt="">
```
Comment (AbdallahGot) [inline aboutus.html]: why the alternative is empty.
 write a descriptive alt like a "background-image".

File: aboutus.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+padding: 20px;
+text-align: center;
+
+}
+.about_p{
+        font-size: 20px;
+    padding: 20px;
+
+}
+@media screen and (max-width: 600px) {
```
Comment (AbdallahGot) [inline aboutus.css]: change this line to be @media screen and (max-width: 768px)
just change the max-width .

**Review by AbdallahGot — COMMENTED** [review]
I think the scroll at top of the page is not necessary.
and try to remove the scroll of the page. 
image

**Comment by sajaabusafia** [comment]
> I think the scroll at top of the page is not necessary.
> and try to remove the scroll of the page.
> image

This part depends on the index.html file, I am going to fix it once it's upload.

**Review by AbdallahGot — APPROVED** [review]

File: aboutus.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./aboutus.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="about_text">
```
Comment (sara219) [inline aboutus.html]: use semantic HTML element since we handling with HTML5

File: aboutus.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <p class="about_text_title">-Our Mission-</p>
+        <p class="about_p" style="text-align: center;">
```
Comment (sara219) [inline aboutus.html]: don't use inline style if you already have a stylesheet

File: aboutus.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            To provide the most actionable Real Estate Website.
+        </p>
+        <p class="about_text_title">About</p>
+
+        <p class="about_p">
```
Comment (sara219) [inline aboutus.html]: `about_p` What this class name means

**Always use descriptive class names, that tell us what is the content**

File: aboutus.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            As a team, we wanted to make the process of getting your own place easier, so we focused on making the browser simple so that the user would not have to figure how it work first, In addition to providing all the services that the user would need.
+        </p>
```
Comment (sara219) [inline aboutus.html:49]: use prettier correctly to format your code  

Peek 2021-03-02 19-15

File: aboutus.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+font-size: 20px;
+font-weight: bold; 
+
+}
+.about_p{
+        font-size: 16px;
+        padding: 15px;
+        font-weight: bold;
+
+}
```
Comment (sara219) [inline aboutus.css]: pay attention to the code formating

File: aboutus.css [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.header{
+    width: 100%;
+}
+#header{
+    width: auto;
+}
```
Comment (sara219) [inline aboutus.css]: there's no header on your HTML page, why you add that rule set!

File: aboutus.css [hunk 9]
```diff
@@ -0,0 +1,60 @@
+*{
+    margin: 0px;
+    margin-top: -3px;
```
Comment (sara219) [inline aboutus.css]: unnecessary code.
no change will happen if you delete it or keep it
( delete it )

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**